client/src/endpoint/orderbook.rs

```diff
@@ -0,0 +1,127 @@
+//! Contains the endpoint for fetching the orderbook for a given asset pair
+use error::Result;
+use std::str::FromStr;
+use stellar_resources::{AssetIdentifier, Orderbook};
+use super::{Body, IntoRequest, Limit};
+use http::{Request, Uri};
+
+/// Given an asset pair, the endpoint will return all bids and asks with an optional
+/// limit parameter to cap the number of records returned.
+///
+/// <https://www.stellar.org/developers/horizon/reference/endpoints/orderbook-details.html>
+///
+/// ## Example
+/// ```
+/// use stellar_client::sync::Client;
+/// use stellar_client::endpoint::{orderbook, trades, Limit};
+///
+/// let client = Client::horizon_test().unwrap();
+///
+/// // Grab a trade so we can get two valid assets.
+/// # let endpoint = trades::All::default().with_limit(1);
```

Comment:
Why do we want to hide this?

Reply:
well i figured these lines had nothing to do with the usage of the orderbook endpoint.  It's all set up and tear down.  Not passionate and can totally include the rest.

Reply:
There is a comment that describes it. But, it's also useful to show the setup here as it demonstrates relationships between endpoints.